region/brinstar/blue.json

```diff
@@ -128,8 +139,21 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canPassBombPassages" ],
-              "unlock": null
+              "requires": [
+                {"or":[
+                  "canPassBombPassages",
+                  {"and":[
+                    "canUsePowerBombs",
+                    {"canShineCharge": {
+                      "usedTiles": 21,
+                      "shinesparkFrames": 0
+                    }},
+                    "canMockball"
+                  ]}
+                ]}
+              ],
+              "unlock": null,
+              "note": "FIXME The Power Bombs in the mockball strat are to clear the obstacle between this node and node 4, to open up room to charge the blue suit. FIXME It's 21 tiles, but with an open end. FIXME Only morph is required if the blocks have already been cleared."
```

Comment:
Sounds like it's all about a future implementation of obstacles for the sake of saving 1 PB.

Reply:
It's not just about saving 1 PB. You could come in from the left with no bombs at all, grab the first item, and leave. But without tracking the bomb blocks properly, the logic can't represent it.